memecity.game/Game/Generate/Item/ItemGenerator.cpp

```diff
@@ -0,0 +1,28 @@
+#include "ItemGenerator.h"
+#include "..\..\Components.h"
+#include "..\..\..\Assets.h"
+
+namespace generate {
+	//place to make all items that needed to be generated when the world is loading.
+	void ItemGenerator::MakeAllItems() {
```

Comment:
lege method?

Reply:
voor nu wel maar hier komen alle items in die we op de grond willen hebben om op te rapen.